lib/rules.js

```diff
@@ -222,9 +237,42 @@ const combiningDiacriticsBlockExceptAcuteAndUmlautAndTildeRule = codepointRuleGe
     .filter(codepoint => codepoint !== acuteAccentCombining)
     .filter(codepoint => codepoint !== umlautCombining)
     .filter(codepoint => codepoint !== tildeCombining),
-  'combining diacritics block except acute and umlaut and ñ'
+  'combining diacritics block except acute and umlaut and tilde'
 );
 
+const combiningDiacriticsBlockExceptAcuteAndUmlautAndNTildeRule = new FunctionRule((s1, s2) => {
+  let s1prime = normalize(s1, 'NFD');
+  let s2prime = normalize(s2, 'NFD');
+
+  [
+    ...combiningDiacriticsBlock
+      .filter(codepoint => codepoint !== acuteAccentCombining)
+      .filter(codepoint => codepoint !== umlautCombining)
+      .filter(codepoint => codepoint !== tildeCombining)
+  ].forEach(codepoint => {
+    s1prime = replaceAll(s1prime, codepoint);
+    s2prime = replaceAll(s2prime, codepoint);
+  });
+
+  // ã
+  s1prime = replaceAll(s1prime, 'a\u0303', 'a');
```

Comment:
The replace above just looks for finds the diacritics and replaces it with empty string, so in unicode if it was a + acute accent, the acute accent would be removed. Since we are targeting specific letter combinations, the third argument allows you to specify the replaced character.